modules/calib3d/src/calibration.cpp

```diff
@@ -47,6 +47,7 @@
 #include "opencv2/calib3d/calib3d_c.h"
 #include <stdio.h>
 #include <iterator>
+#include <iostream>
```

Comment:
most probably is not needed.

Reply:
Indeed: I just wanted to staty close to the 5.x version. I also removed the commented out `cout` code